localfs.js

```diff
@@ -145,7 +160,7 @@ function checkExistingProjects (driver, projects) {
             if (!err) {
                 if (!results[0]) {
                     // let projectOpts = JSON.parse(project.options)
-                    logger.info(`restating ${project.id}}`)
+                    logger.info(`Restarting project ${project.id}}`)
```

Comment:
It's left over from the old version, but it looks like there is an extra `}` on the end

Reply:
Oops - it was the extra `}` that made me spot the typo in `restating`... but then I only fixed the typo